package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "asap",
+  "version": "1.0.0",
+  "description": "HueHueHue",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
Consider adding the author's name in the author field. This provides information about who is responsible for the project. [medium]